Detectors/TPC/base/include/TPCBase/CalArray.h

```diff
@@ -267,6 +270,18 @@ inline const CalArray<T>& CalArray<T>::operator/=(const CalArray<T>& other)
   return *this;
 }
 
+//______________________________________________________________________________
+template <class T>
+inline bool CalArray<T>::operator==(const CalArray<T>& other) const
+{
+  if (!((mPadSubset == other.mPadSubset) && (mPadSubsetNumber == other.mPadSubsetNumber))) {
+    LOG(error) << "pad subset type of the objects it not compatible";
+    return false;
+  }
+  bool isSame = mData == other.mData;
```

Comment:
Readability braces.
```suggestion
  bool isSame = (mData == other.mData);
```

Reply:
Thanks, should I fix it in this PR?
Otherwise I would create a new PR with some other minor changes and fix it there...

Reply:
I thinks this can be fixed in a separate PR. It would be good to get these modification in soon, I think.

Reply:
The FST fails due to the alien-token problem on the builder, but this PR cannot affect the FST, shall I merge this?

Reply:
From my side it can be merged